### ISSUE TITLE
iOS IPV6 issue?

### ISSUE THREAD

**Issue by alamboley** [playgameservices/play-games-plugin-for-unity#1393]
Hello, I sent a build with TestFlight and it was rejected by Apple due to IPV6 issue. Later the client asked me to remove all references to Google Play Games and use directly iOS Game Center, I submitted a new build and it has been approved!

So does someone else had troubles with IPV6 and this plugin? Even if I didn't use it anymore for iOS yet, it's important to fix this issue. Managing IPV6 issues on big project is hard!

For more informations with logs, see my [discussion](http://forum.unity3d.com/threads/unity-ios-and-ipv6-support.404938/page-2#post-2795718) on Unity forum.

**Comment by claywilkinson** [comment]
I reviewed the log and discussion on the Unity forum, but I didn't see any discussion about Google Play Games?  Was it Apple that asked you to remove Google Play Games, or someone else?

Thanks for sharing this!

**Comment by alamboley** [comment]
No Apple doesn't ask me to remove Google Play Games. They didn't give me any clue concerning the issue with IPV6. I tried the iOS setup to run the game locally with IPV6, it was working fine.

Finally the client asks me to remove Play Games for iOS, because he doesn't want a connexion when starting the game, and use GameCenter directly. I submitted back to Apple, and it was accepted. Not a lot of things changed during that time... so I'm wondering if there is a game with Google Play Games which has been approved by Apple recently?
